ui/admin/app/templates/scopes/scope/roles/role/manage-scopes/manage-org-projects.hbs

```diff
@@ -0,0 +1,56 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+{{page-title
+  (t
+    'resources.role.scope.messages.manage-org-projects.title'
+    orgDisplayName=@model.orgScope.displayName
+  )
+}}
+{{#if @model.role.scope.isGlobal}}
+  <Breadcrumbs::Item
+    @text={{t 'resources.role.scope.messages.manage-custom-scopes.title'}}
+    @route='scopes.scope.roles.role.manage-scopes.manage-custom-scopes'
+  />
+{{/if}}
+<Breadcrumbs::Item
+  @text={{t
+    'resources.role.scope.messages.manage-org-projects.title'
+    orgDisplayName=@model.orgScope.displayName
+  }}
+  @route='scopes.scope.roles.role.manage-scopes.manage-org-projects'
+  @model={{@model}}
+/>
```

Comment:
Should this be an `if/else`?

Reply:
Just an `if` since we want to show an additional breadcrum if its a global level role since this page is nested under "Manage Custom Scopes" only in the global level.